docs/en/03_examples/00_elementalarea_with_dataobject.md

```diff
@@ -1,16 +1,12 @@
-# Using ElementalArea with DataObject
+# Using `ElementalArea` with `DataObject`
```

Comment:
I'm not sure if the docs site uses the first heading or the file name in `[CHILDREN]` block cards when there's no title metadata.... To reduce ambiguity please add a metadata title. e.g:

```text
---
title: Using ElementalArea with DataObject
---
```

Optionally add a `summary` and an `icon` in the metadata as well.

Please apply this to all pages.

Reply:
Updated